src/test/java/org/jclouds/vcloud/director/v1_5/compute/strategy/VCloudDirectorComputeServiceAdapterLiveTest.java

```diff
@@ -58,12 +58,17 @@ public void testCreateNodeWithGroupEncodedIntoNameThenStoreCredentials() {
       String name = "vm-" + new Random().nextInt();
 
       Template template = templateBuilder
-              .imageNameMatches("centos6.4x64") // TAI
+              .locationId("https://emea01.canopy-cloud.com/api/vdc/e931a09d-131f-4aaa-a667-efbe02eba428") // TAI2.0
```

Comment:
agreed, I should use probably `-Dtest.jclouds-vcloud-director.template` and remove those hard-coded values.